modules/db_instance/variables.tf

```diff
@@ -346,6 +346,12 @@ variable "restore_to_point_in_time" {
   default     = null
 }
 
+variable "network_type" {
+  description = "The type of network stack"
+  type        = string
+  default     = "IPV4"
```

Comment:
```suggestion
  default     = null
```

Reply:
This has been changed. I'll leave it up to your review to mark these conversations as resolved.